packages/desktop-client/src/components/reports/Header.js

```diff
@@ -86,6 +93,41 @@ function Header({
           gap: 15,
         }}
       >
+        {title === 'Category Spending' && (
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏could we instead use a new prop for this? The title is subject to change, so we might forget to update it here and hence break the logic.

I would recommend: `headerPrefixItems?: React.ChildrenNode`.